src/cmd/cmd.zig

```diff
@@ -1,26 +1,64 @@
 const std = @import("std");
-const cli = @import("zig-cli");
 const base58 = @import("base58-zig");
-const enumFromName = @import("../utils/types.zig").enumFromName;
-const getOrInitIdentity = @import("./helpers.zig").getOrInitIdentity;
-const ContactInfo = @import("../gossip/data.zig").ContactInfo;
-const SOCKET_TAG_GOSSIP = @import("../gossip/data.zig").SOCKET_TAG_GOSSIP;
-const Logger = @import("../trace/log.zig").Logger;
-const Level = @import("../trace/level.zig").Level;
-const io = std.io;
-const Pubkey = @import("../core/pubkey.zig").Pubkey;
-const SocketAddr = @import("../net/net.zig").SocketAddr;
-const echo = @import("../net/echo.zig");
-const GossipService = @import("../gossip/service.zig").GossipService;
-const servePrometheus = @import("../prometheus/http.zig").servePrometheus;
-const globalRegistry = @import("../prometheus/registry.zig").globalRegistry;
-const Registry = @import("../prometheus/registry.zig").Registry;
-const getWallclockMs = @import("../gossip/data.zig").getWallclockMs;
+const cli = @import("zig-cli");
+const network = @import("zig-network");
+const sig = @import("../lib.zig");
```

Comment:
thats a pretty nice way to do imports

Reply:
I've been trying to establish a rough set of rules in my mind about how to structure it. I'm not sure about some edge cases, but the general idea is to group it into a few sections. A section is just a list of imports without any blank lines, and sections are delimited by a blank line, with each section internally alphabetized (sort of).

These are basically the sections I've been using:
1. All the `@import()` calls. No other area of the file should call `@import()` except tests. This section should only import entire libraries from the top-level, including the current project as a single import. I've been structuring this section so `std` is first and the current project is last. Otherwise the middle imports are alphabetized.
2. Nested namespaces, for example `const bincode = sig.bincode`
3. Types from the standard library and external libraries
4. Types from the current project
5. Functions from the standard library and external libraries
6. Functions from the current project
7. Constants from the standard library and external libraries
8. Constants from the current project

I would not expect a file to have all sections. I feel like 3-5 sections is more typical.

Sections 1, 3, 5 and 7 are actually each like multiple sections, but when I split them apart, it looks a bit too cluttered. I've been torn on whether they should be fully alphabetized or if standard library imports should come first.

I know in rust the convention is different. First they group by standard library, then crates, then the current project. Types, functions, and namespaces are all mixed together. But at least in zig, for some reason I'm drawn to separating these things from each other, and I'm not too bothered by the standard library mingling with external libraries.